params/config_arbitrum.go

```diff
@@ -29,8 +29,8 @@ type ArbitrumChainParams struct {
 	InitialArbOSVersion       uint64
 	InitialChainOwner         common.Address
 	GenesisBlockNum           uint64
-	MaxCodeSize               uint64 // Maximum bytecode to permit for a contract. 0 value implies params.MaxCodeSize
-	MaxInitCodeSize           uint64 // Maximum initcode to permit in a creation transaction and create instructions. o value implies params.MaxInitCodeSize
+	MaxCodeSize               uint64 `json:"maxCodeSize,omitempty,"`    // Maximum bytecode to permit for a contract. 0 value implies params.MaxCodeSize
+	MaxInitCodeSize           uint64 `json:"maxInitCodeSize,omitempty"` // Maximum initcode to permit in a creation transaction and create instructions. 0 value implies params.MaxInitCodeSize
```

Comment:
Let's capitalize these JSON field names to match the others (they're capitalized as per the Go field name by default)

Reply:
also, there's an extra comma on the first JSON attribute. I doubt it matters, but probably better not to have it

Reply:
should remove extra comma after `omitempty`

Reply:
Cool! my bad for the typo